features/ProductNews.feature

```diff
@@ -0,0 +1,54 @@
+@owner:derek
+@author:lilla
+
+Feature: Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario: Product news updates can be filtered by type
```

Comment:
In a scenario where you are talking about things being filterable, good detail to have is the user going through with the filtering. This is important in QA cause we want to not only test what a user sees, but the functionality that they use. This could be turned into a scenario outline with a `When` statement around them selecting one of the drop down options and a `Then` around what happens.